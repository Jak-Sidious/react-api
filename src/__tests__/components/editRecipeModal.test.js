import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ModalEditRec from '../../components/commonComponents/editRecipeModal';

describe('ModalEditRec component', () => {
  const del = jest.fn();
  const wrapper = shallow(<ModalEditRec  />);

  it('renders properly without crashing', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('has clickable button', () => {
    expect(wrapper.find('#recEdit').length).toBe(1);
  });
});
