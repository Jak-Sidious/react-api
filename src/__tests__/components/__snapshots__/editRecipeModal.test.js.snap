// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalEditRec component renders properly without crashing 1`] = `
<Modal
  closeIcon={true}
  closeOnDimmerClick={true}
  closeOnDocumentClick={false}
  dimmer={true}
  eventPool="Modal"
>
  <ModalHeader>
    Edit Recipe
  </ModalHeader>
  <ModalContent>
    <Form
      as="form"
    >
      <FormInput
        as={[Function]}
        control={[Function]}
        label="Recipe name"
        name="recipeName"
      />
      <FormTextArea
        as={[Function]}
        control={[Function]}
        label="Recipe Description"
        name="recipeDesc"
      />
      <Button
        as="button"
        color="blue"
        id="recEdit"
      >
        Save
      </Button>
    </Form>
  </ModalContent>
</Modal>
`;
